packer/linux/conf/bin/bk-install-elastic-stack.sh

```diff
@@ -100,6 +100,11 @@ if [[ "${BUILDKITE_AGENT_ENABLE_GIT_MIRRORS_EXPERIMENT}" == "true" ]] ; then
   fi
 fi
 
+# If the agent token path is set, use that instead of BUILDKITE_AGENT_TOKEN
+if [[ -z "${BUILDKITE_AGENT_TOKEN_PATH}" ]] ; then
+    BUILDKITE_AGENT_TOKEN="$(aws ssm get-parameter --name ${BUILDKITE_AGENT_TOKEN_PATH} --with-decryption | jq -r .Parameter.Value)"
```

Comment:
You can use `--query` instead of needing `jq` here:

```suggestion
    BUILDKITE_AGENT_TOKEN="$(aws ssm get-parameter --name "${BUILDKITE_AGENT_TOKEN_PATH}" --with-decryption --query Parameter.Value"
```